AmmadAmjad-CodeTest/Server/controllers/db.js

```diff
@@ -0,0 +1,49 @@
+const Sequelize = require('sequelize');
+
+var sequelize ;
+
+
+if(process.env.NODE_ENV === 'test'){
+  sequelize =  new Sequelize(
+    null,
+    null,
+    null, {
+      dialect: 'sqlite',
+      storage: './data/db-test.sqlite',
+      operatorsAliases: false,
+      logging: false
+  }); 
+}else{
+  sequelize =  new Sequelize(
+    "hacker-news",
```

Comment:
It would be good to put these into a environment variables file. You can think about a way to manage the configurations.